spec/index.bs

```diff
@@ -8612,6 +8612,10 @@ will affect a render pass's {{GPURenderPassDescriptor/depthStencilAttachment}}:
     - If |descriptor|.{{GPUDepthStencilState/stencilFront}} or
             |descriptor|.{{GPUDepthStencilState/stencilBack}} are not the default values:
             - |descriptor|.{{GPUDepthStencilState/format}} must have a stencil component.
+    - If |descriptor|.{{GPUDepthStencilState/format}} has a depth component:
+        - |descriptor|.{{GPUDepthStencilState/depthWriteEnabled}} must be [=map/exist|provided=].
+        - |descriptor|.{{GPUDepthStencilState/depthCompare}} must be [=map/exist|provided=].
+
```

Comment:
@kainino0x mentioned in https://github.com/gpuweb/gpuweb/issues/3905#issuecomment-1476882031 that there's likely an additional relaxation of the requirements that can be done here if depth comparison isn't used. However I don't recall if that was part of the editors agreement mentioned in his next post.

In any case, this is step in the right direction and does not prevent that additional logic from being implemented later on, so I'm OK with this landing as-is and only bring up the other comment to aid in future spec archaeology.

Reply:
> However I don't recall if that was part of the editors agreement mentioned in his next post.

It was. Sorry, comment was unclear.